services/src/main/java/org/keycloak/services/resources/admin/info/ServerInfoAdminResource.java

```diff
@@ -113,7 +113,7 @@ public ServerInfoRepresentation getInfo() {
                                 Collectors.toMap(
                                         ClientSignatureVerifierProvider::isAsymmetricAlgorithm,
                                         clientSignatureVerifier -> Collections.singletonList(clientSignatureVerifier.getAlgorithm()),
-                                        (l1, l2) -> listCombiner(l1, l2)
+                                        (l1, l2) -> ServerInfoAdminResource.<String>listCombiner(l1, l2)
```

Comment:
Sorry about the nitpick. It works too. 

However, we introduce a warning because the type can be inferred too. What if we just expand this lambda to an anonymous class? By removing the warning it also helps to avoid accidentaly removing the type because of the warning.

Reply:
Thanks @pedroigor no problem! though I wonder would following alternative work without warning?

```diff
diff --git a/services/src/main/java/org/keycloak/services/resources/admin/info/ServerInfoAdminResource.java b/services/src/main/java/org/keycloak/services/resources/admin/info/ServerInfoAdminResource.java
index 9b66c7f8be..a23bf90c0c 100755
--- a/services/src/main/java/org/keycloak/services/resources/admin/info/ServerInfoAdminResource.java
+++ b/services/src/main/java/org/keycloak/services/resources/admin/info/ServerInfoAdminResource.java
@@ -113,10 +113,12 @@ public class ServerInfoAdminResource {
                                 Collectors.toMap(
                                         ClientSignatureVerifierProvider::isAsymmetricAlgorithm,
                                         clientSignatureVerifier -> Collections.singletonList(clientSignatureVerifier.getAlgorithm()),
-                                        (l1, l2) -> listCombiner(l1, l2)
-                                                .stream()
-                                                .sorted()
-                                                .collect(Collectors.toList()),
+                                        (l1, l2) -> {
+                                            List<String> result = listCombiner(l1, l2);
+                                            return result.stream()
+                                                    .sorted()
+                                                    .collect(Collectors.toList());
+                                        },
                                         HashMap::new
                                 )
                         );
```

BTW how / where could I see the warning? I did not find it while compiling on command line. I've also got sonarlint and checkstyle but I did not seem to get the warning that it could be inferred.

Reply:
I see the warning in my InteliJ not really when running a java compiler. The syntax above also works for me ...

I hope you see my point ... Just want to avoid someone reverting your change because the type `<String>` is marked as unnecessary (at least in IntelliJ which a lot of people are using here). Refactoring the code to a lambda is a more drastic change ....

A nitpick, otherwise we can just merge it as is.